benchexec/tools/svf_svc.py

```diff
@@ -40,23 +38,20 @@ def cmdline(self, executable, options, task, rlimits):
 
         if task.property_file:
             options += ["--prop", task.property_file]
+
+        if rlimits.cputime:
+            options += ["--time-limit", str(rlimits.cputime)]
+
         return [executable] + options + [task.single_input_file]
 
     def determine_result(self, run):
-        good_exit = False
-        verified = True
-
         for line in run.output:
             if line.startswith("Correct"):
-                good_exit |= True
-                verified &= True
+                return result.RESULT_TRUE_PROP
             elif line.startswith("Incorrect"):
-                good_exit |= True
-                verified &= False
-
-        if not good_exit:
-            return result.RESULT_UNKNOWN
-        elif verified:
-            return result.RESULT_TRUE_PROP
-        else:
-            return result.RESULT_FALSE_PROP
+                return result.RESULT_FALSE_PROP
+            elif line.startswith("Error"):
+                # The line should contain the error message with details.
+                return line
```

Comment:
For consistency with other tools and BenchExec itself it would be good if the result here would conform to the same structure as is typical: Either return plain `RESULT_ERROR` or return `RESULT_ERROR + "(" + reason + ")"`.

Furthermore, the reason should not be too long, because otherwise the tables become impractical to use. Something like "exception", "segmentation fault", "parsing failed", etc. as the reason is good and really useful, but something like `Exception in thread "main" java.util.NoSuchElementException: No value present` would be too long in practice. Is the "error message with details" expected to be reasonably short in practice?